modules/shapefile/src/lib/parse-dbf.js

```diff
@@ -0,0 +1,141 @@
+/* global TextDecoder */
+
+const LITTLE_ENDIAN = true;
+
+export default function parseDbf(arrayBuffer, options) {
```

Comment:
This file looks very clean, nicely done. Hopefully we can later make it streaming without complicating it too much...